test/e2e/common/node/container_probe.go

```diff
@@ -213,7 +214,10 @@ var _ = SIGDescribe("Probing container", func() {
 		Testname: Pod liveness probe, container exec timeout, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	framework.ConformanceIt("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
```

Comment:
Does it also make sense to remove NodeConformance?

Reply:
@chewong nope. this is still useful in `NodeConformance`.

Reply:
we are also checking the flag now. so should work fine.